.github/workflows/build-ios.yml

```diff
@@ -157,6 +157,11 @@ jobs:
           path: 'breez-sdk-liquid'
           ref: ${{ needs.setup.outputs.liquid-sdk-ref }}
 
+      - name: Use production .podspec
+        if: ${{ needs.setup.outputs.use-published-plugins == 'false' }}
+        working-directory: breez-sdk-liquid/packages/flutter/ios/
+        run: mv flutter_breez_liquid.podspec.production flutter_breez_liquid.podspec
```

Comment:
I find this confusing because only the production podspec can download the published plugin (from what I see)

Reply:
It would be easy to catch faulty builds where there was an issue and published framework is downloaded instead.

Each generated code has it's own content hash, if the plugin's content hash & framework's content has does not match, it will cause runtime errors.

There is one case where this would be hard to catch. The content hashes may match if there wasn't a change on bindings itself.

Reply:
I just fine the combination with `use-published-plugins` confusing. It's fine though

Reply:
> I just fine the combination with `use-published-plugins` confusing. It's fine though

Published plugins do use production .podspec and since we're not building them from scratch, this step is not needed when using a published plugin.